src/main/java/com/forgeessentials/permissions/core/ZonedPermissionHelper.java

```diff
@@ -769,6 +769,16 @@ else if (context.getSourceLocationStart() != null)
                 loc = new WorldPoint(dim, context.getSourceLocationStart());
         }
 
+        if (context instanceof PlayerContext)
```

Comment:
@spacebuilder2020 This is how it is in v1.12 but here it throws an error because PlayerContext does not exist. Any idea what we can do here instead of checking for PlayerContext?

Reply:
The changes here can be restored to the stuff in 745 since the PlayerContext used on 1.7.10 lives in the Forge Essentials repo and if the sender is a player the player object is set..